content/influxdb/cloud-iox/get-started/query.md

```diff
@@ -0,0 +1,358 @@
+---
+title: Get started querying data
+seotitle: Query data | Get started with InfluxDB
+list_title: Query data
+description: >
+  Get started querying data in InfluxDB by learning about Flux and InfluxQL and
+  using tools like the InfluxDB UI, `influx` CLI, and InfluxDB API.
+menu:
+  influxdb_cloud_iox:
+    name: Query data
+    parent: Get started
+    identifier: get-started-query-data
+weight: 102
+metadata: [3 / 3]
+related:
+  - /influxdb/cloud-iox/query-data/
+---
+
+InfluxDB Cloud backed by InfluxDB IOx supports multiple query languages:
+
+- **SQL**: Traditional SQL powered by the [Apache Arrow DataFusion](https://arrow.apache.org/datafusion/)
+  query engine. The supported SQL syntax is similar to PostgreSQL.
```

Comment:
I believe it's FlightSQL, not Postgres

Reply:
Flight SQL is the client-server protocol used to query Apache Arrow data with SQL. It's more of an API than a query language. The point of this note is to give users something familiar to compare the syntax to.